oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -554,13 +604,20 @@ int DumpHandler::fileAckWithMetaData(uint8_t /*fileStatus*/,
                                      uint32_t /*metaDataValue3*/,
                                      uint32_t /*metaDataValue4*/)
 {
+    info("File Handle in fileAckWithMetaData is {FILEHANDLE}", "FILEHANDLE",
```

Comment:
Can we remove this debug trace ?

Reply:
Keeping these debug traces until new dump support is stable from all the components. Will remove it later on.